espnet/nets/chainer_backend/asr_transformer/__init__.py

```diff
@@ -0,0 +1,4 @@
+from espnet.nets.chainer_backend.asr_transformer.e2e import E2E  # NOQA
```

Comment:
I'm not a big fan of making `asr_transformer` directory. I think `asr` directory should be OK.